procrastinate/contrib/django/admin.py

```diff
@@ -106,12 +119,19 @@ def short_task_name(self, instance: models.ProcrastinateJob) -> str:
 
     @admin.display(description="Args")
     def pretty_args(self, instance: models.ProcrastinateJob) -> str:
-        indent = 2 if len(instance.args) > 1 or len(str(instance.args)) > 30 else None
-        pretty_json = json.dumps(instance.args, indent=indent)
-        if len(pretty_json) > 2000:
-            pretty_json = pretty_json[:2000] + "..."
+        arg_rows = [
+            format_html(
+                "<tr><td>{key}</td><td>{value}</td></tr>",
+                key=key,
+                value=ellipsize_middle(value),
+            )
+            for key, value in instance.args.items()
```

Comment:
What about calls with thousands of short args ? Shouldn't we limit that too ? Maybe more than 30 args don't make a lot of sense to display ?

Reply:
It seems to me kind of contrived that you would define a function with thousands of parameters.

Reply:
Well that could be:
```
@app.task
def x(**kwargs):
    ...
```

Then it's only up to how big the dict is.